packages/@ember/helper/type-tests/set-helper-manager.test.ts

```diff
@@ -41,7 +42,7 @@ export const SIMPLE_CLASSIC_HELPER_MANAGER = new SimpleClassicHelperManager();
 
 expectTypeOf(
   setHelperManager(() => SIMPLE_CLASSIC_HELPER_MANAGER, Wrapper.prototype)
-).toEqualTypeOf<Wrapper>();
+).toEqualTypeOf<Wrapper<any, any, any>>();
```

Comment:
Can this just specify the types?

```suggestion
).toEqualTypeOf<Wrapper<unknown, unknown[], Dict<unknown>>>();
```

Reply:
That doesn't work unfortunately. I don't know exactly why.

Reply:
Huh. I can pull it and take a look after merging.

Reply:
If you're curious: it's because referencing `Wrapper.prototype` ends up causing TS to resolve the relevant type of `Wrapper.prototype` when passed to `setHelperManager` as `Wrapper<any, any, any>`. You can explicitly cast it *there*, but that obviously doesn't help. (I *thought* that microsoft/TypeScript#47607 might be able make it work, but nope: that doesn't support this particular pattern.)